App/Redux/LogInReducer.ts

```diff
@@ -0,0 +1,76 @@
+/// <reference path="../../typings/tsd.d.ts" />
+import { ISettings, IUserProfile, ISettingsInfo, ISettingsLists} from './LoginActions';
+
+export interface ISettingsState{
+  account: string;
+  project: string;
+  team: string;
+}
+
+const initialState: ISettingsState = {
+    account: 'acc1',
+    project: 'proj1',
+    team: 'team1',
+};
+
+export function updateDefaultState(state: ISettingsState = initialState, action: ISettings): ISettingsState {
+  switch (action.type) {
+    case 'DEFAULT_SETTINGS':
```

Comment:
lets not call the action as Default settings since we are going to use it in general right on any selection?